content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -683,6 +687,62 @@ More configuration options can be found on the [kong-plugin-ddtrace][3] plugin d
 [3]: https://github.com/DataDog/kong-plugin-ddtrace#configuration
 
 {{% /tab %}}
+
+{{% tab "Apache HTTPd" %}}
+
+## HTTPd with Datadog module
+Datadog provides a [module][1] for enhancing [Apache HTTP Server][2] capabilities with APM Tracing.
+
+### Module installation
+<div class="alert alert-warning">
+  <strong>Note:</strong> Only Apache HTTP Server 2.4.x is supported.
+</div>
+
+The module is provided as a shared library for dynamical loading by HTTPd. Each supported platform
+and architecture has its own artifact hosted on [httpd-datadog's repository][1].
+
+Run the following script to download the latest version of the module:
+
+```bash
+get_latest_release() {
+  curl --silent "https://api.github.com/repos/$1/releases/latest" | jq --raw-output .tag_name
+}
+
+RELEASE_TAG=$(get_latest_release DataDog/httpd-datadog)
+wget "https://github.com/DataDog/httpd-datadog/releases/download/$RELEASE_TAG/linux-x86_64-mod_datadog.tar.gz"
+```
+
+When unpacking the tarball, the resulting file is `mod_datadog.so`, the shared library that must
+be loaded by the server.
+
+Place it in the directory where HTTPd searches for modules, typically `/usr/local/apache2/modules`.
+
+Load the module by adding the following line in the configuration file:
+
+```nginx
+LoadModule datadog_module modules/mod_datadog.so
+```
+
+To enable the module, make sure to restart or reload HTTPd.
+
+### Module configuration
+By default, all requests are traced and sent to the Datadog Agent.
+To change the module default behaviour, use `Datadog*` directives described in the Datadog module's [API documentation][3].
+
+For example, the following configuration sets the service name to `my-service` and the sampling rate to 10%
+
+```nginx
+LoadModule datadog_module modules/mod_datadog.so
+
+DatadogServiceName my-app
+DatadogSamplingRate 0.1
+```
+
+[1]: https://github.com/DataDog/httpd-datadog
+[2]: https://httpd.apache.org/
+[3]: https://github.com/DataDog/httpd-datadog/blob/dmehala/add-trace/doc/configuration.md
```

Comment:
[q] Anything blocking us from merging this in?

Reply:
Yes. I still need to make some adjustment.